Packs/ADPITAdmin/Integrations/ADP_IT_Admin/README.md

```diff
@@ -0,0 +1,583 @@
+ADP integration to import ADP user data
+This integration was integrated and tested with version xx of ADP IT Admin
```

Comment:
replace the xx with the actual version of the product

Reply:
Done